python/dgl/graphbolt/minibatch_sampler.py

```diff
@@ -108,11 +100,72 @@ class MinibatchSampler(IterDataPipe):
     >>> data_pipe = data_pipe.map(add_one)
     >>> list(data_pipe)
     [tensor([1, 2, 3, 4]), tensor([5, 6, 7, 8]), tensor([ 9, 10])]
+
+    7. Heterogeneous node/edge IDs.
```

Comment:
Heterogeneous node IDs.

edge IDs is not recommended in our new design, let's use node ID in example.

Reply:
what about homogeneous edge IDs? not recommended either?

Reply:
No, not recommended either.

Reply:
why eid is not recommended in graphbolt?